readme.md

```diff
@@ -54,7 +61,12 @@ console.log(results);
 
 Your Answer:
 ```text
-
+where is the AuthorModel from? it did not specify where it's coming from. We can fix the following way:
+1. var results = db.AuthorModel.find(...);
+2. //use mongoose
+var mongoose =require('mongoose');
+mongoose.connect('mongodb://url');
+var results=mongoose.AuthorModel.find({...});
```

Comment:
While you're correct here, the reason we're not seeing results logged has to do with promises